unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GoToGenesisPlazaHUD/GoToGenesisPlazaHUDView.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.GoToGenesisPlazaHUD
+{
+    public class GoToGenesisPlazaHUDView : MonoBehaviour
+    {
+        public bool isOpen { get; private set; } = false;
+
+        public event System.Action OnClose;
+
+        private const string PATH = "GoToGenesisPlazaHUD";
+        private const string VIEW_OBJECT_NAME = "_GoToGenesisPlazaHUD";
+
+        [SerializeField] private ShowHideAnimator GoToGenesisPlazaAnimator;
```

Comment:
```suggestion
        [SerializeField] private ShowHideAnimator goToGenesisPlazaAnimator;
```
If you add `[FormerlySerializedAs("GoToGenesisPlazaAnimator")]` save the project and then remove it, the serialized fields stay in place.

Reply:
Done! Thanks!